adbfunctions.sh

```diff
@@ -180,7 +180,7 @@ getpackagename() {
 function adblogcat()
 {
     ANDROID_DEVICE=$1
-    LOGFILE1=xxx_$ANDROID_SERIAL
```

Comment:
this is beautiful!

Reply:
Actually, I'll accept this. The next several lines were hidden. Since the logs are saved to at least a file with serial number, that's good enough for me.